packages/components/src/components/primitives/Tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Tooltip, Button } from '..';
+import { fireEvent, screen, waitFor } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
+import { renderWithProviders } from 'tests/renderWithProviders';
+import { calculateTooltipPlacement } from '.';
+
+describe('Tooltip', () => {
+  test('it should render', async () => {
+    renderWithProviders(
+      <Tooltip label="Test!" side="bottom" positioning="center">
+        <Button>Hover me!</Button>
+      </Tooltip>
+    );
+
+    act(() => {
+      fireEvent.mouseOver(screen.getByRole('button'));
+    });
+
+    await waitFor(() =>
+      expect(screen.queryByTestId('cmpsr.tooltip')).toBeTruthy()
+    );
+  });
+
+  test('it should render children', async () => {
```

Comment:
Is async needed?

Reply:
Here it is needed because the tooltip is not rendered immediately after hovering the button, and the test will fail if we don't wait for the tooltip to be shown.